### ISSUE TITLE
PSX Games Not Found

### ISSUE THREAD

**Issue by cspmosh** [muldjord/skyscraper#35]
Probably a newbie problem but most of my PSX games cannot find a match using the "Skyscraper -p psx --region US" command. For instance, can't find Mega Man 8 (USA). I've looked at screenscraper.fr for example and verified the games are on there and have tried renaming local files to different things for various regions with and without parentheses and changing roman numerals but it won't pick up. I haven't had this issue with other platforms.

Also, love what you've done. You have the best scraper.

**Comment by muldjord** [comment]
"--region US" won't work. It has to be "--region us". Can you provide the
full filename with suffix for the file? I can't really test it just with
"Mega Man 8 (USA)"

Thanks.

**Comment by cspmosh** [comment]
Sorry, that was a typo. I did use lowercase 'us' for the region. Not sure if it's using the zip to match the file name or the files inside. Zip file is 'Mega Man 8 (USA).zip'. Inside the zip I have 'Mega Man 8 (USA) (Track 1).bin', 'Mega Man 8 (USA) (Track 2).bin', 'Mega Man 8 (USA) (Track 3).bin' and 'Mega Man 8 (USA).cue'

**Comment by cspmosh** [comment]
Also, the only reason I used the --region switch is because it didn't work without it and I saw the default was eu and thought it might be trying to find a different name when scraping based on region.

**Comment by muldjord** [comment]
It uses either the filename, the sha1 checksum or the md5 checksum. I doubt your zip file is recognized with the checksums, so it will have to be recognized from the filename. I'll look into that.

**Comment by muldjord** [comment]
I just looked at your command line again. You need to provide a scraping module aswell, otherwise it tries to scrape from the data it has cached locally so far. But if you haven't scraped the games before, it won't find anything. So please try with 'Skyscraper -p psx -s screenscraper --updatedb' and see if that returns anything.

**Comment by cspmosh** [comment]
lol no luck but I got this amazing message 
![untitled](https://user-images.githubusercontent.com/40703038/43286853-d464cbe2-90e8-11e8-9447-8ffb2a292a0e.png)

**Comment by cspmosh** [comment]
tbh I haven't even tried running the roms yet so maybe they're bad files. Although I would't think that would matter if it was using the file name to scrape the data. I'm at a loss.

**Comment by muldjord** [comment]
"Mega Man 8 (USA).zip" and "Twisted Metal 2 (USA).zip" is not on screenscraper, so that's why it doesn't find them. Skyscraper doesn't extract any files which is why it isn't recognized. :) I just tried with both files as .cue files and that works just fine.

**Comment by cspmosh** [comment]
Well forgive me for taking up your precious time. Thanks so much for helping. Is there any way to support what you're doing?

**Comment by muldjord** [comment]
You could also try it with "Skyscraper -p psx -s thegamesdb --updatedb". That might work without extracting the files, since thegamesdb uses a name search instead of precise names.

**Comment by muldjord** [comment]
Well, I have a Patreon but don't really use it. I'd suggest just helping out the community in general if you feel like it. :) Thanks for asking.

**Comment by cspmosh** [comment]
thegamesdb found them all... I must not understand how Skyscraper works. I thought it automatically tried pulling from all the sources on the initial pull (without switches). Well thanks man!

**Comment by muldjord** [comment]
No problem! But since thegamesdb changed their api and I updated Skyscraper to support the new one, they have added a request limit, so I had to take it off the list of scrapers it auto-scrapes from. So basically now we're down to only scraping from screenscraper unless you specifically ask it to use 'thegamesdb' as you just did. (Notice the request remaining counter when running it using 'thegamesdb'. If it reaches 0 it will quit. This is a limit they have on their end).
